connect-location/src/main/java/com/ifttt/location/ConnectLocation.java

```diff
@@ -126,28 +193,8 @@ private void setUpPolling() {
         );
     }
 
-    @Override
-    public void onStateChanged(
-        ConnectButtonState currentState, ConnectButtonState previousState, Connection connection
-    ) {
-        if (currentState == ConnectButtonState.Enabled
-            || currentState == ConnectButtonState.Disabled
-            || currentState == ConnectButtonState.Initial) {
-            geofenceProvider.updateGeofences(connection);
-        }
-
-        if (hasEnabledLocationUserFeature(connection) && permissionCallback != null) {
-            permissionCallback.onRequestLocationPermission();
-        }
-    }
-
-    @Override
-    public void onError(ErrorResponse errorResponse) {
-        // No-op
-    }
-
     @CheckResult
-    private static boolean hasEnabledLocationUserFeature(Connection connection) {
+    static boolean hasEnabledLocationUserFeature(Connection connection) {
```

Comment:
intentional to make it package private?

Reply:
Yeah, we use that in the separate ButtonStateChangeListener as well.